config/nrfconnect/chip-module/CMakeLists.txt

```diff
@@ -50,21 +50,12 @@ include(${COMMON_CMAKE_SOURCE_DIR}/chip_gn_args.cmake)
 include(${COMMON_CMAKE_SOURCE_DIR}/chip_gn.cmake)
 
 # Prepare compiler flags
-matter_add_flags(-D_DEFAULT_SOURCE)
-
-if (CHIP_CFLAGS)
```

Comment:
Is this removal intentional?

Reply:
Discussed F2F and brought the change back.